style.css

```diff
@@ -15,57 +17,152 @@
     display: flex;
     flex-direction: column;
     align-items: center;
-    gap: 2rem;
+    gap: calc(1.5rem + 2vw);
 }
 
 .intro-title {
-    font-size: 3.5rem;
+    font-size: calc(1.5rem + 2.5vw);
     line-height: 1.2em;
+    padding: 2vh;
     text-align: center;
-    margin-top: 2rem;
+    margin-top: calc(1rem + 1vw);
 }
 
 .intro-subtitle {
-    font-size: 1.5rem;
+    font-size: calc(1rem + 0.5vw);
     text-align: center;
-    padding:0 5rem;
 }
 
 .nav {
     display: flex;
     width: 100%;
-    background-color: #F4653E;
-    color: #fefefe;
+    background-color: var(--primary-color);
+    color: var(--font-white);
     align-items: center;
-    padding: 0 5rem;
+    padding: 0 calc(1rem + 4vw);
 }
 
 .nav-title {
     display: flex;
-    align-items: baseline;
+    align-items: center;
+    padding: 0.5rem;
+    width: 100%;
+}
+
+.nav-title .material-icons {
+    margin-left: auto;
+    cursor: pointer;
+}
+
+.nav-title .brand {
+    display: flex;
+    align-items:  baseline;
     gap: 0.5rem;
 }
 
 .links {
+    position: absolute;
+    left: 0;
     display: flex;
-    margin-left: auto;
-    gap: 1rem;
+    width: 100vw;
```

Comment:
Just a suggestion. You can also use 
```
left:0;
right:0;
````
to achieve the same result by using positioning only

Reply:
I used position here to get the elements out of the screen in case of mobile view so that on click of menu button it moves down. 
I used mobile first approach here.

you're saying that I should do 
.links {
    position: absolute;
    left: 0;
    right: 0;
}
right?

Reply:
I understood your approach and I was just suggesting an alternate approach nothing wrong with what you are doing right now. I thought since you are using positioning already you can also use it to resize the element.

Reply:
I got you and I would love to try that but this didn't work. Can you share the code or a codesandbox link?

Reply:
It's a very small change actually. Check this   https://codesandbox.io/s/naughty-dubinsky-9vbhr?file=/style.css:1111-1120

Reply:
Thanks I got it😄